package.json

```diff
@@ -1,47 +1,49 @@
 {
```

Comment:
**suggestion:** can we revert the whitespace changes in this file, as it's noisy and will make the history hard to read.

Reply:
Oh weirdly this is still showing as the whole file being edited for me. It looks like we're still using tabs at the top level?